Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -1810,3 +1810,12 @@ calcExecutionTime transporterConfig' autopayPaymentStage scheduledAt = do
     Just DDF.NOTIFICATION_ATTEMPTING -> addUTCTime executionTimeDiff scheduledAt
     Just DDF.EXECUTION_SCHEDULED -> addUTCTime executionTimeDiff scheduledAt
     _ -> scheduledAt
+
+updateOperatingCity :: (EsqDBFlow m r, EsqDBReplicaFlow m r, CacheFlow m r, MonadFlow m) => (Id SP.Person, Id DM.Merchant, Id DMOC.MerchantOperatingCity) -> Context.City -> m APISuccess
+updateOperatingCity (personId, merchantId, _) city = do
+  merchant <- runInReplica $ CQM.findById merchantId >>= fromMaybeM (MerchantNotFound merchantId.getId)
+  cityId <- runInReplica $ CQMOC.getMerchantOpCityId Nothing merchant (Just city)
```

Comment:
why runInReplica for cached queries

Reply:
oversight on my part. Although this PR is not required anymore so closing it